src/index.js

```diff
@@ -88,10 +88,21 @@ class MixpanelTool {
     if (this.state.record && this.isRequestValid(requestObject)) {
       let properties = {};
       try {
-        let urlParams = this.getUrlParams(requestObject);
+        let urlParams;
+        if (requestObject.request.method === "POST") {
+          urlParams = getQueryStringParams(requestObject.request.postData.text);
+        } else {
```

Comment:
@sachinmour : else if(requestObject.request.method === "GET") {}  will be better for understanding.